defaults/main.yml

```diff
@@ -49,7 +49,11 @@ systemd_timer_consul_service_id: '{{ systemd_timer_consul_service_name }}'
 systemd_timer_consul_check_path: '/usr/local/bin/check-timer-status.sh'
 systemd_timer_consul_address: '{{ ansible_local.wireguard.vpn_ip }}'
 systemd_timer_consul_warning: false
-
+systemd_timer_consul_success_before_passing: 0
+# Default value if the Timer doesn't restart else value based on the timeout divided by consul interval check
+systemd_timer_failure_success_before_warning: '{{ 1 if systemd_timer_restart == "no" else (systemd_timer_timeout_sec / 30) }}'
```

Comment:
If there's no variable for `consul-service` to set `interval` then we need to add it.